src/Symfony/Component/Form/Tests/AbstractLayoutTest.php

```diff
@@ -113,6 +113,8 @@ protected function assertWidgetMatchesXpath(FormView $view, array $vars, $xpath)
 
     abstract protected function renderLabel(FormView $view, $label = null, array $vars = array());
 
+    abstract protected function renderHelp(FormView $view);
```

Comment:
To keep compat with branch 3.4 (which is broken because of this, as spotted by the deps=high failure), we cannot add a new abstract method here.
Here is what we did on 2.8 in a similar case:
https://github.com/mpiot/symfony/blob/2.8/src/Symfony/Component/Form/Tests/AbstractLayoutTest.php#L114

Should apply here also.

Reply:
Thank you :-)